src/main/java/io/github/opencubicchunks/cubicchunks/core/client/ClientEventHandler.java

```diff
@@ -223,6 +226,8 @@ public void mouseReleased(int mouseX, int mouseY) {
         private static final int MORE_WORLD_OPTIONS = 3;
 
         private static final int CC_ENABLE_BUTTON_ID = 11;
+        private static final List<ResourceLocation> LIST_OF_GEN_OPTIONS = new ArrayList<ResourceLocation>();
+        private static int CURRENT_GEN_OPTION = 0;
```

Comment:
Set this to the correct value from config when the GUI is initialized, to avoid config and GUI not being in sync.